### ISSUE TITLE
Organisation vs Organization

### ISSUE THREAD

**Issue by millingw** [ckan/ckan#7988]
## CKAN version
2.10

## Describe the bug
CKAN appears to default to the American spelling of Organization for organization urls, e.g. https://find.researchdata.scot/organization/adr. As we are deploying into a UK-based service, we'd like to consistently refer to organizations using British English, i.e. organisation instead of organization so https://find.researchdata.scot/organisation/adr . Is there a straightforward way to do this, e.g. via some configuration or locale setting, or would this involve a global search / replace of all instances of organization in the code base?

**Comment by amercader** [comment]
Yes, there is a specific `en_GB` locale that you can set as default in the configuration:
```
ckan.locale_default = en_GB

```
You might want to tweak the `ckan.locales_filtered_out` and `ckan.locale_order` as well if you want to offer American English as an option in your languages list.
